build.gradle

```diff
@@ -131,8 +131,8 @@ allprojects {
     jar.preserveFileTimestamps = false
```

Comment:
We also need to remove jdk16 from `def dockerJdkVariants = [ "jdk17", "jdk16", ]`

Reply:
Line 503 of build.gradle @siladu

Reply:
done